test/suites/examples/install-karpenter.yaml

```diff
@@ -2,51 +2,46 @@
 apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
-  name: install-karpenter-task
-  namespace: tekton-tests
+  name: install-karpenter-release-eks-task
+  namespace: karpenter-tests
 spec:
   description: |
     Install Karpenter and necessary resources onto an EKS cluster.
   params:
-  - name: cluster-name
-    description: The name of the cluster to install Karpenter
+  - name: test-cluster-name
+    description: The name of the EKS cluster to install Karpenter
   - name: karpenter-version
     description: The version of Karpenter to install. Needs to be in vx.y.z format where x, y, and z are integers.
   - name: account-id
     description: Account ID where to deploy resources
   - name: pre-install-image
     description: Docker image for the Tekton pod to install resources. This needs minimally needs awscli, kubectl, and curl.
-  - name: aws-region
-    description: AWS Region where cluster is deployed
-  workspaces:
-  - name: config
-    description: |
-      A workspace into which a kubeconfig file called `kubeconfig` will be written that will contain the information required to access the cluster.
   steps:
   - name: karpenter-pre-install
     image: $(params.pre-install-image)
     script: |
-        aws eks update-kubeconfig --name $(params.cluster-name) --region $(params.aws-region)
+        export AWS_REGION=$(http://169.254.169.254/latest/meta-data/placement/region)
+        aws eks update-kubeconfig --name $(params.test-cluster-name) --region ${AWS_REGION}
```

Comment:
Let's use the SDK default region. Either via ~/aws/config or the env var AWS_DEFAULT_REGION. We shouldn't encourage cross region testing.